CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Fixed
 
+- Improved font rendering
```

Comment:
Could you reference the pull request like in the rest of the changelog?

Reply:
Added PR ref